lib/ansible/module_utils/netapp.py

```diff
@@ -104,6 +114,34 @@ def create_sf_connection(module, port=None):
         module.fail_json(msg="the python SolidFire SDK module is required")
 
 
+def setup_na_ontap_zapi(module, vserver=None):
+    hostname = module.params['hostname']
+    username = module.params['username']
+    password = module.params['password']
+    https = module.params['https']
+
+    if HAS_NETAPP_LIB:
+        # set up zapi
+        server = zapi.NaServer(hostname)
+        server.set_username(username)
+        server.set_password(password)
+        if vserver:
+            server.set_vserver(vserver)
+        # Todo : Replace hard-coded values with configurable parameters.
+        server.set_api_version(major=1, minor=21)
+        # default is HTTPS
+        if https in ['True', 'true']:
```

Comment:
Can this option be passed through the module? I don't see the parameter in the example below. Also can it be modified into a type="boolean" instead of type="str"

Reply:
I think we had some issue when it was a boolean, but let me try again and see if it works.

Because we give a default on line 89 it only needs to be in an example where https is set to true.